sklearn/random_projection.py

```diff
@@ -436,15 +439,16 @@ class GaussianRandomProjection(BaseRandomProjection):
         very conservative estimated of the required number of components
         as it makes no assumption on the structure of the dataset.
 
-    eps : strictly positive float, optional (default=0.1)
+    eps : float, default=0.1
+        Strictly positive.
```

Comment:
Add it inside the description as shown above

Reply:
Would this be better?
`eps : float, default=0.1
        Parameter to control the quality of the embedding according to
        the Johnson-Lindenstrauss lemma when n_components is set to
        'auto'. Needs to be strictly positive.
`